app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less

```diff
@@ -419,6 +419,15 @@
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
     .minicart-wrapper {
         margin-top: @indent__s;
+        .product {
```

Comment:
Please apply clearer for parent block when using `float` properties.
```
.lib-clearfix();
```

Reply:
Fixed it, thanks!